src/lib/customPropTypes.js

```diff
@@ -181,6 +181,49 @@ export const demand = (requiredProps) => {
 }
 
 /**
+ * Ensure a component can render as a node passed as a prop value in place of children.
+ */
+export const content = (...args) => every([
+  disallow(['children']),
+  PropTypes.node,
+])(...args)
+
+/**
+ * Item shorthand is a description of a component that can be a literal,
+ * a props object, or an element.
+ */
+export const item = (...args) => every([
+  disallow(['children']),
+  PropTypes.oneOfType([
+    PropTypes.element,
+    PropTypes.number,
+    PropTypes.object,
+    PropTypes.string,
+  ]),
+])(...args)
+
+/**
+ * ArrayOf shorthand ensures a prop is an array of item shorthand
+ */
+export const items = (...args) => every([
+  disallow(['children']),
```

Comment:
This is duplicate disallowing since the item disallows.  Suggest removing that.  I'm also not sure we should have this checker.  Seems we could just do `PropTypes.arrayOf(item)` in the component and `collection` below to simplify.  Will let you make the call on that.